data/pc/latest/proto.yml

```diff
@@ -1761,8 +1761,11 @@
       block_interaction_type: varint
       small_explosion_particle: Particle
       large_explosion_particle: Particle
-      explosion_sound_name: string
-      sound_range?: f32
+      soundId: varint
+      _: soundId ?
+         if 0:
+            soundName: string
+            range?: f32
```

Comment:
This is the same logic that is used for packet_entity_sound_effect, maybe a common type can be made to avoid duplication